dev-packages/application-manager/src/generator/frontend-generator.ts

```diff
@@ -111,9 +111,9 @@ const electron = require('electron');
 const { join, resolve } = require('path');
 const { isMaster } = require('cluster');
 const { fork } = require('child_process');
-const { app, BrowserWindow, ipcMain, Menu } = electron;
+const { app, shell, BrowserWindow, ipcMain, Menu } = electron;
 
-const applicationName = \`${this.pck.props.frontend.config.applicationName}\`;
+const applicationName = '${this.pck.props.frontend.config.applicationName}';
```

Comment:
Why? Please do not include irrelevant changes.

If I use the `What's wrong with this application name` application name, I will have the following generated code:
```js
const applicationName = 'What's wrong with this application name';
```

It has compiler errors.

Reply:
Well, if I want to call the application `${fork}` with the old string delimiters there would also be the an issue. Thanks for pointing one more corner case.

This only mean that we should escape whatever is passed, as safety.